metatensor-core/CHANGELOG.md

```diff
@@ -0,0 +1,74 @@
+# Changelog
+
+All notable changes to metatensor-core will be documented in this file.
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### metatensor-core C++
+
+#### Added
+
+#### Fixed
+
+#### Changed
+
+#### Removed
+
+
+### metatensor-core C
+
+#### Added
+
+#### Fixed
+
+#### Changed
+
+#### Removed
+
+
+### metatensor-core Python
+
+#### Added
+
+#### Fixed
+
+#### Changed
+
+#### Removed
```

Comment:
Do we really want it to be that detailed? If yes, the rust part missing, right?

Reply:
I'd say yes, we want to document all changes in public API. This is also why the rust part is not here. For metatensor-core, using Rust is an implementation detail.